### ISSUE TITLE
incorporate presentation into website

### ISSUE THREAD

**Issue by ghost** [capstone-coal/pycoal#13]
It was [mentioned](https://groups.google.com/forum/#!topic/coal-capstone/32zUxQbeFKQ) that the presentation should be added to the website. We will be dubbing voiceovers and creating a multimedia file in addition to the raw slides which will be ready after we submit our report Wednesday. Should the presentation be uploaded as-is, or should its contents be incorporated into the website?

The [infrared image](https://www.nasa.gov/multimedia/imagegallery/image_feature_681.html) in slide 2 was found by running the capstone-coal avatar --
![capstone-coal avatar](https://avatars1.githubusercontent.com/u/22801081?v=3&s=200)
-- through the reverse image search [TinEye](http://tineye.com). I would propose using this as the website background image for a consistent visual identity.

**Comment by lewismc** [comment]
ack. I agree. @xiaomei7 can you please take care of this? Thank you

**Comment by lewismc** [comment]
@xiaomei7 can you please look into this? Thank you

**Comment by lewismc** [comment]
@xiaomei7 ping

**Comment by lewismc** [comment]
@xiaomei7 the background image for the Website can now be found [here](https://github.com/capstone-coal/capstone-coal.github.io/blob/master/img/161344main_image_feature_681b_ys_full.jpg), the code to embed it into the Website can be found at 
https://github.com/capstone-coal/capstone-coal.github.io/blob/master/css/stylesheet.css#L12
Please update the background thank you.

**Comment by lewismc** [comment]
This is now addressed thank you @xiaomei7
